nodetool

```diff
@@ -49,7 +49,7 @@ main([ConfigFile]) ->
 
     Action   = strip(get_arg("action",   Args)),
     Node     = strip(get_arg("node",     Args)),
-    NameType = get_arg("nametype", Args, shortnames),
+    NameType = list_to_atom2(get_arg("nametype", Args, "shortnames")),
```

Comment:
No need to be defensive. You can use `list_to_existing_atom` directly.
The try/catch defeats the original purpose of the function.
I believe those two atoms (`longnames` and `shortnames`) already exist. If not, don't worry too much about this. `list_to_atom` will do just fine for this escript.